cmd/virt-manifest/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM centos:7
+
+# The jmliger-virt7-upstream-epel-7 has updated libvirt binaries. Since
+# the statically linked libvirt-go code requires a runtime version of libvirt
+# no-lower than what was used when the binary was built.
+RUN yum -y install wget && \
+    wget -P /etc/yum.repos.d https://copr.fedorainfracloud.org/coprs/jmliger/virt7-upstream/repo/epel-7/jmliger-virt7-upstream-epel-7.repo  \
+    && yum -y install libvirt-client \
+    && wget https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm \
+    && yum -y install epel-release-latest-7.noarch.rpm \
+    && yum -y install xmlstarlet \
```

Comment:
I believe this is still necessary. We're using the same static libvirt client libraries as virt-handler. Attempting to remove them results in:

/virt-manifest: /lib64/libvirt.so.0: version `LIBVIRT_2.2.0' not found (required by /virt-manifest)